Jetchat/gradle/libs.versions.toml

```diff
@@ -8,6 +8,7 @@ androidx-benchmark-junit4 = "1.1.0-beta04"
 androidx-constraintlayout = "1.0.1"
 androidx-corektx = "1.8.0"
 androidx-lifecycle-compose = "2.5.1"
+androidx-lifecycle-runtime-compose = "2.6.0-alpha01"
```

Comment:
Which version of Compose does this bring in? :) because we are aiming to depend on compose stable on main

Reply:
+--- androidx.lifecycle:lifecycle-runtime-compose:2.6.0-alpha01
|    +--- androidx.compose.ui:ui:1.0.1 -> 1.2.1 (*)

Reply:
Perfect, good to go!